test/Makefile

```diff
@@ -94,6 +94,8 @@ coverage: check-go-version $(COVERAGE_BINARIES) $(TEST_PAYLOAD)
 	$(MAKE) -C ../crit/ unit-test GOFLAGS="-coverprofile=${COVERAGE_PATH}/coverprofile-crit-unit-test"
 	$(MAKE) -C crit/ e2e-test GOCOVERDIR=${COVERAGE_PATH}
 	$(MAKE) -C crit/ clean
+	# Generate coverage for ../scripts
+	$(MAKE) -C ../scripts/ test GOFLAGS="-coverprofile=${COVERAGE_PATH}/coverprofile-scripts-unit-test" GOCOVERDIR="${COVERAGE_PATH}"
```

Comment:
I believe the `GOCOVERDIR` variable is not needed for `go test -coverprofile=...`
https://go.dev/blog/integration-test-coverage#using-the-integration-test-to-collect-coverage-data